docs/actors/development/continuous_integration.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Continuous integration
+description: Learn how to set up automated builds, deploys, and testing for your actors using GitHub Actions or Bitbucket Pipelines.
+paths:
+    - actors/development/continuous-integration
+---
+
+# Continuous integration for actors
+
+If you have a project consisting of several actors, or even one actor that requires frequent updates, you may want to automate some of the development process. Instead of manually pushing your code, building each actor, then testing it, you could perform the whole process whenever you run `git push`.
+
+You can automate actor builds and tests using your Git repository's automated workflows like [GitHub Actions](https://github.com/features/actions) or [Bitbucket Pipelines](https://bitbucket.org/product/features/pipelines).
+
+This article will focus on GitHub but [we also have a guide for Bitbucket](https://help.apify.com/en/articles/1861038-setting-up-continuous-integration-for-apify-actors-on-bitbucket).
+
+## TL;DR
+
+Below is an example GitHub Actions workflow that will run your tests and build your actor every time you push your code to GitHub. This workflow supports both **latest** and **beta** builds. Copy the code into **separate files** in your actor repo's **.github/workflows** directory: e.g. **.github/workflows/main.yml** and **.github/workflows/beta.yml**.
+
+Each time you push to the **main/master** branch, a new **latest** version of your actor will be built. When you push to **develop**, a **beta** version will be built.
+
+```marked-tabs
```

Comment:
I am not 100% sure about naming main and beta, what about  to name it by branches main and develop or latest and beta

Reply:
Do you mean like this?
```
Each time you push to the **main** branch, a new **main** version of your actor will be built. When you push to **develop**, a **develop** version will be built.
```

Reply:
I mean the filenames in workflows and multi-file editor.

Reply:
Ah, I see! Sure thing!